lib/id_token.js

```diff
@@ -29,10 +29,17 @@ var claims = {
     `https://${jwt.payload.iss}` === idp.issuer // google
   ,
   // audience - is the token intended for me?
-  audience: (jwt, app) =>
+  audience: (jwt, app) => {
     // audience - should be your OAuth app ID
-    jwt.payload.aud === app
```

Comment:
This is how it was fixed in Grant:
```js
[].concat(jwt.payload.aud).includes(app)
```

Reply:
Damn, i'm getting rusty on all the JS typeless magic...